<ui version="4.0" >
 <class>equationEditor</class>
 <widget class="QDialog" name="equationEditor" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>571</width>
    <height>458</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Equation properties</string>
  </property>
  <widget class="QTabWidget" name="pdeTabs" >
   <property name="geometry" >
    <rect>
     <x>210</x>
     <y>60</y>
     <width>351</width>
     <height>391</height>
    </rect>
   </property>
   <property name="tabShape" >
    <enum>QTabWidget::Rounded</enum>
   </property>
   <property name="currentIndex" >
    <number>0</number>
   </property>
   <property name="elideMode" >
    <enum>Qt::ElideNone</enum>
   </property>
   <widget class="QWidget" name="heatEquation" >
    <property name="accessibleName" >
     <string/>
    </property>
    <attribute name="title" >
     <string>Heat equation</string>
    </attribute>
    <widget class="QGroupBox" name="heatEquationConvectionGroup" >
     <property name="geometry" >
      <rect>
       <x>10</x>
       <y>10</y>
       <width>321</width>
       <height>91</height>
      </rect>
     </property>
     <property name="title" >
      <string>Convection</string>
     </property>
     <widget class="QRadioButton" name="heatEquationConvectionNone" >
      <property name="geometry" >
       <rect>
        <x>10</x>
        <y>20</y>
        <width>83</width>
        <height>18</height>
       </rect>
      </property>
      <property name="text" >
       <string>None</string>
      </property>
      <property name="checked" >
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QRadioButton" name="heatEquationConvectionConstant" >
      <property name="enabled" >
       <bool>true</bool>
      </property>
      <property name="geometry" >
       <rect>
        <x>10</x>
        <y>40</y>
        <width>83</width>
        <height>18</height>
       </rect>
      </property>
      <property name="text" >
       <string>Constant</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="heatEquationConvectionComputed" >
      <property name="geometry" >
       <rect>
        <x>10</x>
        <y>60</y>
        <width>83</width>
        <height>18</height>
       </rect>
      </property>
      <property name="text" >
       <string>Computed</string>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="heatEquationPCMGroup" >
     <property name="geometry" >
      <rect>
       <x>10</x>
       <y>110</y>
       <width>321</width>
       <height>141</height>
      </rect>
     </property>
     <property name="title" >
      <string>Phase change model</string>
     </property>
     <widget class="QRadioButton" name="heatEquationPCMNone" >
      <property name="geometry" >
       <rect>
        <x>10</x>
        <y>20</y>
        <width>83</width>
        <height>18</height>
       </rect>
      </property>
      <property name="text" >
       <string>None</string>
      </property>
      <property name="checked" >
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QRadioButton" name="heatEquationPCMSpatial1" >
      <property name="geometry" >
       <rect>
        <x>10</x>
        <y>40</y>
        <width>83</width>
        <height>18</height>
       </rect>
      </property>
      <property name="text" >
       <string>Spatial 1</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="heatEquationPCMSpatial2" >
      <property name="geometry" >
       <rect>
        <x>10</x>
        <y>60</y>
        <width>83</width>
        <height>18</height>
       </rect>
      </property>
      <property name="text" >
       <string>Spatial 2</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="heatEquationPCMTemporal" >
      <property name="geometry" >
       <rect>
        <x>10</x>
        <y>80</y>
        <width>83</width>
        <height>18</height>
       </rect>
      </property>
      <property name="text" >
       <string>Temporal</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="heatEquationPCMLatentHeatRelease" >
      <property name="geometry" >
       <rect>
        <x>10</x>
        <y>110</y>
        <width>131</width>
        <height>18</height>
       </rect>
      </property>
      <property name="text" >
       <string>Latent heat release</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QWidget" name="linearElasticity" >
    <attribute name="title" >
     <string>Linear elasticity</string>
    </attribute>
    <widget class="QCheckBox" name="linearElasticityPlaneStress" >
     <property name="geometry" >
      <rect>
       <x>10</x>
       <y>10</y>
       <width>91</width>
       <height>18</height>
      </rect>
     </property>
     <property name="text" >
      <string>Plane stress</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="navierStokes" >
    <attribute name="title" >
     <string>Navier-Stokes</string>
    </attribute>
    <widget class="QCheckBox" name="navierStokesCalculateHydrostaticPressure" >
     <property name="geometry" >
      <rect>
       <x>10</x>
       <y>10</y>
       <width>211</width>
       <height>18</height>
      </rect>
     </property>
     <property name="text" >
      <string>Calculate hydrostatic pressure</string>
     </property>
    </widget>
    <widget class="QGroupBox" name="navierStokesTurbulenceGroup" >
     <property name="geometry" >
      <rect>
       <x>10</x>
       <y>40</y>
       <width>321</width>
       <height>101</height>
      </rect>
     </property>
     <property name="title" >
      <string>Turbulence model</string>
     </property>
     <widget class="QRadioButton" name="navierStokesTurbulenceModelNone" >
      <property name="geometry" >
       <rect>
        <x>10</x>
        <y>20</y>
        <width>83</width>
        <height>18</height>
       </rect>
      </property>
      <property name="text" >
       <string>None</string>
      </property>
      <property name="checked" >
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QRadioButton" name="navierStokesTurbulenceModelKE" >
      <property name="geometry" >
       <rect>
        <x>10</x>
        <y>40</y>
        <width>83</width>
        <height>18</height>
       </rect>
      </property>
      <property name="text" >
       <string>KE</string>
      </property>
     </widget>
     <widget class="QLabel" name="navierStokesKEClipLabel" >
      <property name="geometry" >
       <rect>
        <x>10</x>
        <y>70</y>
        <width>41</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text" >
       <string>KE clip:</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="navierStokesKEClipEdit" >
      <property name="geometry" >
       <rect>
        <x>60</x>
        <y>70</y>
        <width>251</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text" >
       <string>1.0e-6</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QWidget" name="tab" >
    <attribute name="title" >
     <string>Advection diffusion</string>
    </attribute>
    <widget class="QGroupBox" name="advectionDiffusionConvectionGroup" >
     <property name="geometry" >
      <rect>
       <x>10</x>
       <y>10</y>
       <width>321</width>
       <height>91</height>
      </rect>
     </property>
     <property name="title" >
      <string>Convection</string>
     </property>
     <widget class="QRadioButton" name="advectionDiffusionConvectionNone" >
      <property name="geometry" >
       <rect>
        <x>10</x>
        <y>20</y>
        <width>83</width>
        <height>18</height>
       </rect>
      </property>
      <property name="text" >
       <string>None</string>
      </property>
      <property name="checked" >
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QRadioButton" name="advectionDiffusionConvectionConstant" >
      <property name="geometry" >
       <rect>
        <x>10</x>
        <y>40</y>
        <width>83</width>
        <height>18</height>
       </rect>
      </property>
      <property name="text" >
       <string>Constant</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="advectionDiffusionConvectionComputed" >
      <property name="geometry" >
       <rect>
        <x>10</x>
        <y>60</y>
        <width>83</width>
        <height>18</height>
       </rect>
      </property>
      <property name="text" >
       <string>Computed</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QWidget" name="helmholtzEquation" >
    <attribute name="title" >
     <string>Helmholtz equation</string>
    </attribute>
    <widget class="QLabel" name="helmholtzEquationAngularFrequencylabel" >
     <property name="geometry" >
      <rect>
       <x>10</x>
       <y>10</y>
       <width>101</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text" >
      <string>Angular frequency:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="helmholtzEquationAngularFrequencyEdit" >
     <property name="geometry" >
      <rect>
       <x>120</x>
       <y>10</y>
       <width>211</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QLabel" name="equationLabel" >
   <property name="geometry" >
    <rect>
     <x>10</x>
     <y>20</y>
     <width>51</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text" >
    <string>Equation:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="equationNameEdit" >
   <property name="geometry" >
    <rect>
     <x>70</x>
     <y>20</y>
     <width>321</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QGroupBox" name="activeSolversGroup" >
   <property name="geometry" >
    <rect>
     <x>10</x>
     <y>60</y>
     <width>191</width>
     <height>391</height>
    </rect>
   </property>
   <property name="title" >
    <string>Active solvers</string>
   </property>
   <widget class="QCheckBox" name="heatEquationActive" >
    <property name="geometry" >
     <rect>
      <x>10</x>
      <y>20</y>
      <width>141</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text" >
     <string>Heat equation</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="linearElasticityActive" >
    <property name="geometry" >
     <rect>
      <x>10</x>
      <y>40</y>
      <width>141</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text" >
     <string>Linear elasticity</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="navierStokesActive" >
    <property name="geometry" >
     <rect>
      <x>10</x>
      <y>60</y>
      <width>101</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text" >
     <string>Navier-Stokes</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="helmholtzEquationActive" >
    <property name="geometry" >
     <rect>
      <x>10</x>
      <y>100</y>
      <width>121</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text" >
     <string>Helmholtz equation</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="advectionDiffusionActive" >
    <property name="geometry" >
     <rect>
      <x>10</x>
      <y>80</y>
      <width>131</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text" >
     <string>Advection-diffusion</string>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="acceptEquation" >
   <property name="geometry" >
    <rect>
     <x>400</x>
     <y>20</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text" >
    <string>Ok</string>
   </property>
  </widget>
  <widget class="QPushButton" name="deleteEquation" >
   <property name="geometry" >
    <rect>
     <x>480</x>
     <y>20</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text" >
    <string>Delete</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
